src/components/DayPicker.jsx

```diff
@@ -68,11 +72,17 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY',
 };
 
+const triggerInitialVisibleMonth = (props) => (
+  props.hidden ? moment() : props.initialVisibleMonth()
```

Comment:
@ljharb asked for it to be broken out in to a seperate function in case we need the logic elsewhere, not 100% where that would be.

@majapw let me know which you prefer, getting conflicting suggestions :)